pegen/pegen.c

```diff
@@ -693,3 +704,27 @@ store_name(Parser *p, expr_ty load_name)
                     Store,
                     EXTRA_EXPR(load_name, load_name));
 }
+
+expr_ty
+del_name(Parser *p, expr_ty load_name)
+{
+    return _Py_Name(load_name->v.Name.id,
+                    Del,
+                    EXTRA_EXPR(load_name, load_name));
+}
+
+asdl_seq *
+targets_map_to_del_name(Parser *p, asdl_seq *seq)
```

Comment:
Same comment on naming here.

Also I wonder if we should start adding a brief block comment to each helper function explaining what it does. (Then again maybe just changing the name is sufficient for now.)